.github/workflows/SSW.SophieBot.yml

```diff
@@ -0,0 +1,152 @@
+name: BuildAndDeploy
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - src/SSW.SophieBot/**
+      - .github/workflows/SSW.SophieBot.yml
+      - .github/workflows/Build-LUIS.ps1
+      - .github/workflows/Build-Orchestrator.ps1
+      - .github/workflows/LUUtils.ps1
+      - .github/workflows/PublishUtils.ps1
+      - .github/workflows/Publish-WebApp.ps1
+
+concurrency:
```

Comment:
Why do we need this?

Reply:
These two ps scripts are for publishing to Azure app service. They mainly do appsettings.json merging.

Reply:
Sorry I was asking why we need concurrency setting

Reply:
Concurrency setting is for ensuring only one workflow per env will run at a time